packages/components/src/tools-panel/test/index.js

```diff
@@ -425,4 +451,64 @@ describe( 'ToolsPanel', () => {
 			expect( altMenuItem ).toHaveAttribute( 'aria-checked', 'false' );
 		} );
 	} );
+
+	describe( 'rendering via SlotFills', () => {
+		beforeEach( () => {
+			jest.clearAllMocks();
+			controlProps.attributes.value = true;
```

Comment:
Should we also reset `controlProps.attributes.value = true;` _after_ this suite of tests has run?

Reply:
Thanks. I've moved this to run for the entire suite of tests and removed the unneeded clearing of mocks for that block of tests.